src/test/scala/uk/gov/nationalarchives/aws/utils/SNSUtilsTest.scala

```diff
@@ -0,0 +1,23 @@
+package uk.gov.nationalarchives.aws.utils
+
+import org.mockito.{ArgumentCaptor, Mockito, MockitoSugar}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers._
+import software.amazon.awssdk.services.sns.SnsClient
+import software.amazon.awssdk.services.sns.model.{PublishRequest, PublishResponse}
+
+class SNSUtilsTest extends AnyFlatSpec with MockitoSugar  {
+  "The publish method" should "be called with the correct parameters" in {
+    val snsClient = Mockito.mock(classOf[SnsClient])
+    val snsUtils = SNSUtils(snsClient)
+    val argumentCaptor: ArgumentCaptor[PublishRequest] = ArgumentCaptor.forClass(classOf[PublishRequest])
+    val mockResponse = PublishResponse.builder().build()
+
+    doAnswer(() => mockResponse).when(snsClient).publish(argumentCaptor.capture())
+
+    snsUtils.publish("test message", "testTopicArn")
```

Comment:
Is it worth extracting the strings "test message" and "testTopicArn" into variables as they're used twice? I know the other tests duplicate the strings and it's personal preference so feel free to ignore me.